ext/intl/tests/bug75090.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+Bug #75090 Constants of parent IntlCalendar class not inherited
+--SKIPIF--
+<?php if( !extension_loaded( 'intl' ) ) print 'skip'; ?>
+--FILE--
+<?php
+class Foo extends \IntlCalendar {}
+
+$fooReflection = new \ReflectionClass(\Foo::class);
+$intlGregorianCalendarReflection = new \ReflectionClass(\IntlGregorianCalendar::class);
+
+printf("%d constants in %s, ", count($fooReflection->getConstants()), \Foo::class);
+if ($fooReflection->getParentClass()) {
+    printf("%s extends %s\n", Foo::class, $fooReflection->getParentClass()->getName());
+}
+printf("%d constants in %s, ", count($intlGregorianCalendarReflection->getConstants()), \IntlGregorianCalendar::class);
+if ($intlGregorianCalendarReflection->getParentClass()) {
+    printf("%s extends %s\n", \IntlGregorianCalendar::class, $intlGregorianCalendarReflection->getParentClass()->getName());
+}
+printf("%d constants in %s\n", count((new \ReflectionClass(\IntlCalendar::class))->getConstants()), \IntlCalendar::class);
+?>
+===DONE===
+--EXPECT--
+39 constants in Foo, Foo extends IntlCalendar
```

Comment:
Some of the constants are conditionally defined based on the ICU version, so testing for a specific number is probably not a good idea.